src/bit-systems/billboard.ts

```diff
@@ -0,0 +1,93 @@
+import { FrameSchedulerSystem } from "aframe";
+import { defineQuery, enterQuery, exitQuery } from "bitecs";
+import { HubsWorld } from "../app";
+import { Billboard } from "../bit-components";
+
+const billboardQuery = defineQuery([Billboard]);
+const billboardEnterQuery = enterQuery(billboardQuery);
+const billboardExitQuery = exitQuery(billboardQuery);
+
+const isMobileVR = AFRAME.utils.device.isMobileVR();
+
+const targetPos = new THREE.Vector3();
```

Comment:
We are trying not to use the `THREE` global in new code, import the individual things you use instead.